examples/configurations/provider-integration-azure/README.md

```diff
@@ -0,0 +1,18 @@
+### Notes
+
+1. Set the required variables in `terraform.tfvars`:
+   ```
+   confluent_cloud_api_key    = "<your-api-key>"
+   confluent_cloud_api_secret = "<your-api-secret>"
+   environment_id             = "env-xxxxx"
+   azure_tenant_id           = "12345678-1234-1234-1234-123456789abc"
+   ```
+
+2. After `terraform apply`, follow the warning instructions to complete Azure setup:
+   - Run the `az ad sp create` command from outputs
+   - Grant permissions in Azure Portal → Enterprise Applications
+   - Re-run `terraform apply` to validate
```

Comment:
I really like the additional effort you put into creating these examples, but I feel they're not very useful on their own unless they are connected to a real usage example, like the Connect or Tableflow examples. For instance:

* https://github.com/confluentinc/terraform-provider-confluent/blob/master/examples/configurations/tableflow/byob-aws-storage/main.tf#L351

* https://github.com/confluentinc/terraform-provider-confluent/blob/master/examples/configurations/connectors/s3-sink-connector-assume-role/main.tf#L442

I would recommend deleting these examples for now and adding new connector or tableflow examples (GCP, Azure) in a separate PR.